app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -138,8 +145,8 @@ export default function GroupChatView() {
     (text) => service.conversations.sendMessage(groupChatId, text),
     {
       onSuccess: () => {
-        queryClient.invalidateQueries(["groupChatMessages", groupChatId]);
-        queryClient.invalidateQueries(["groupChats"]);
+        queryClient.invalidateQueries(groupChatMessagesKey(groupChatId));
+        queryClient.invalidateQueries([groupChatBaseKey]);
```

Comment:
I don't think this should be named like this. Maybe `groupChatsList`?

Reply:
Yeah that's a better name, missed another place where this is used so I'll update there too